### PR TITLE
BETA: Register rl.is-a.dev

### PR DESCRIPTION
**Pull request by aftercares0**
Added `rl.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Comment by Rylazius** [comment]
i want to use this as my bio domain

File: domains/rl.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Rylazius",
+        "email": "nodejsenjoyer@gmail.com"
+    },
+    "record": {
+        "TXT": "8232e838f145c8ffd79d7c4958111c"
```
Comment (SX-9) [inline domains/rl.json:7]: txt alone does nothing

**Review by satr14washere — CHANGES_REQUESTED** [review]

File: domains/rl.json (hunk 1 above)
Comment (Rylazius) [reply domains/rl.json:7]: i forgor wait me a lil bit

**Comment by Rylazius** [comment]
ive changed the txt record to cname

**Review by satr14washere — APPROVED** [review]

**Pull request merged**